### DISCUSSION TITLE
Termux-open fail

### DISCUSSION THREAD

**Discussion by DanRH90** [termux/termux-app#2616] [Q&A] [answered]
Hello,

I had the app installed on another phone and I used it to work remotely from the phone. One thing that I need to do is download pdfs with scp command and open them, I was able to do this through the termux-open command.

I got a new phone and when trying to do this I get an error saying "This file could not be accessed. Check your connection or make the file name shorter." 

Im using the same pdf viewer from the old phone, the files are the same ones so I don't understand what changed. Does anyone have a solution for this? 

Thanks

**Comment by agnostic-apollo** [comment, marked as answer]
Check https://github.com/termux/termux-app/issues/2617#issuecomment-1050258551
